rpc/handlers_test.go

```diff
@@ -1940,6 +1940,58 @@ func TestTransactionStatus(t *testing.T) {
 	}
 }
 
+func TestTransactionNotFoundStatus(t *testing.T) {
```

Comment:
Let's make this a sub-test on the top-level `TestTransactionStatus` test (using `t.Run`).

Reply:
done